theme/templates/accounts/profile.html

```diff
@@ -162,8 +162,8 @@ <h4 class="text-muted">
                                     {# Organization #}
                                     <fieldset class="col-sm-6">
                                         <label>Organization *</label>
-                                        <input class="form-control form-required" {% if profile.organization %}value="{{ profile.organization }}"{% endif %} name="organization">
-                                        <small class="text-muted">The name of the organization you work for.</small>
+                                        <input class="form-control form-required tagsinput" {% if profile.organization %}value="{{ profile.organization }}"{% endif %} name="organization">
+                                        <small class="text-muted">The names of up to two organizations you work for.</small>
```

Comment:
We need to add a `<br>` tag after this small text so that validation error messages display in the next line.